pkg/crc/config/viper_config.go

```diff
@@ -33,17 +32,16 @@ func NewViperStorage(configFile, envPrefix string) (*ViperStorage, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error reading configuration file '%s': %v", configFile, err)
 	}
+	v.WatchConfig()
```

Comment:
we had issues with `WatchConfig()` in the past, see 04832bdd3

Reply:
Yes this is what I try to solve! It works now.

Reply:
Details are needed in the commit log then ;) Mentioning the issue we used to have is very useful for reviewers, as well as explaining why it's no longer a problem (but after seeing again the old log, I guess the bug we had really was because we were misusing `Set()`)

Reply:
I removed the use of Watch.